.github/workflows/pr-test.yaml

```diff
@@ -1,11 +1,10 @@
+name: "Test PR"
+
 on:
-  push:
-    branches:
-      - master
   pull_request:
-    types: [opened, reopened, synchronize, ready_for_review, edited]
+    branches:
+      - "**" # matches every branch
```

Comment:
You don't need these lines. Without a branch specifier, it'll run on PRs against all branches.